<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEquipEnergyItemWidget</class>
 <widget class="QWidget" name="CEquipEnergyItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>45</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>45</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>50</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CEquipEnergyItemWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelIndex">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);
font: 18pt &quot;Agency FB&quot;;
font: bold;</string>
         </property>
         <property name="text">
          <string>0 1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="NameFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelType">
              <property name="styleSheet">
               <string notr="true">font: bold;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>航天器</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelID">
              <property name="minimumSize">
               <size>
                <width>158</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 12pt &quot;新宋体&quot;;
font: bold;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>发现号 OV-103</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>labelID</zorder>
           <zorder>labelType</zorder>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelLogo">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_energy">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>能耗情况</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_load">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>载荷情况</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="EnergyWidget" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="LoadWidget" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_para1">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>268</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_paraA">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>参数A</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_para2">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>97%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_paraB">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>参数B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_para3">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: bold;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>255</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_paraC">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>参数C</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
